packages/errors/src/index.ts

```diff
@@ -0,0 +1,40 @@
+import { ErrorCode } from './error-codes';
+
+export { ErrorCode } from './error-codes';
+
+export class FuelError extends Error {
+  static readonly CODES = ErrorCode;
+
+  static parse(e: unknown) {
+    const error = e as FuelError;
+
+    if (error.code === undefined) {
+      throw new FuelError(ErrorCode.PARSE_FAILED, "missing 'code' property");
+    }
+
+    const enumValues = Object.values(ErrorCode)
+    const codeIsUnknown = enumValues.includes(error.code)
+
+    if (!codeIsUnknown) {
```

Comment:
I think this is on me. 🤦‍♂️ 

```suggestion
    const codeIsUnknown = !enumValues.includes(error.code)

    if (codeIsUnknown) {
```